spec/rubocop/cop/rails/uniq_before_pluck_spec.rb

```diff
@@ -3,55 +3,83 @@
 
 require 'spec_helper'
 
-describe RuboCop::Cop::Rails::UniqBeforePluck do
-  let(:corrected) { 'Model.uniq.pluck(:id)' }
-  subject(:cop) { described_class.new }
+describe RuboCop::Cop::Rails::UniqBeforePluck, :config do
+  subject(:cop) { described_class.new(config) }
 
-  shared_examples_for 'UniqBeforePluck cop' do |source, action|
+  shared_examples_for 'UniqBeforePluck cop' do |method, source, action|
     if action == :correct
```

Comment:
That's reasonable. It's difficult to guess what risk auto-correct poses in this case, so go with that idea.